metrics-0.1.0.json

```diff
@@ -19,39 +19,54 @@
   "description": "This schema defines the structure and values for reporting Q/A metrics for projects.",
```

Comment:
Please also add a description for the `Rating`, mentioning that `A` is good and `F` is bad, to avoid misunderstandings.